File: js-quest06/ex00/my_string_index.js
function my_string_index(haystack, needle) {

    for (i = 0; i < haystack.length; i++) {
         if (haystack[i] == needle) {
             return i;
         }  
    }
     return -1;

}

//console.log(my_string_index("hello", "l"));

File: quest02/ex08/my_string_index.c
#include <stdio.h>

int my_string_index(char* haystack, char needle)
{   
    int i;
    for(i = 0; haystack[i] != '\0'; i++) {
        if (needle == haystack[i]) {
             return i;
        }  
        
    }

    return -1;
}
   
/*int main() {
    printf("%d\n", my_string_index("hello", 'l'));
}*/